digitalstrategie/templatetags/extra_tags.py

```diff
@@ -0,0 +1,11 @@
+import requests
+from django import template
+from django.utils.html import mark_safe
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_external_footer():
+    response = requests.get('https://www.berlin.de/rbmskzl/__i9/ohne/foot.inc')
+    return mark_safe(response.text)
```

Comment:
as discussed we are marking an external file safe, not idea but not sure on alternate way

Reply:
I thought about and think it is fine. Also, there is no other way, I can think of.